x-pack/plugins/fleet/server/services/epm/kibana/assets/install.ts

```diff
@@ -152,18 +153,80 @@ export async function installKibanaAssets(options: {
     managed: true,
   });
 
-  await makeManagedIndexPatternsGlobal(savedObjectsClient);
+  await makeManagedIndexPatternsGlobal(soClient);
 
   const installedAssets = await installKibanaSavedObjects({
     logger,
     savedObjectsImporter,
-    kibanaAssets: assetsToInstall,
+    kibanaAssets: assetsToInstall.filter((asset) => asset.type !== KibanaSavedObjectType.slo),
   });
 
+  const sloAssets = assetsToInstall.filter((asset) => asset.type === KibanaSavedObjectType.slo);
+
+  await installSLOAssets({ sloAssets, sloClient, soClient, esClient, spaceId });
+
   return installedAssets;
```

Comment:
@shahzad31 Don't we want to return here all installed assets including the `sloAssets`?